docs/GettingStarted.md

```diff
@@ -595,3 +595,5 @@ development cycle.  See the [Windows Hardware Developer documentation](https://l
 ## eBPF external extensions
 Extensions that integrate with eBPF for Windows:
 - XDP extension: [xdp-for-windows](https://github.com/microsoft/xdp-for-windows)
+- ntosebpfext extension: [ntosebpfext](https://github.com/microsoft/ntosebpfext)
```

Comment:
For the extensions in the ntos repo, should we be pointing to the code, or the readme for that extension, or what exactly? This first link is the entire github repo itself, but the extension is actually only a subfolder.

Maybe we should link to:
https://github.com/microsoft/ntosebpfext/tree/main/ebpf_extensions/ntosebpfext and then the netevent one should be https://github.com/microsoft/ntosebpfext/tree/main/ebpf_extensions/neteventebpfext?

Reply:
> For the extensions in the ntos repo, should we be pointing to the code, or the readme for that extension, or what exactly?

Currently, the readme for ntosebpfext process creation is tracked in a separate issue. Once that is done, we can always update here. Until then, we are pointing to the main page.